content/en/platform/corda/5.0/introduction/personas.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Personas"
+date: 2023-05-31
+version: 'Corda 5.0'
+menu:
+  corda5:
+    identifier: corda5-personas
+    parent: corda5-intro
+    weight: 4050
+section_menu: corda5
+---
+# Personas
+
+The Corda documentation is written from the perspectives of a collection of people; each with their own problems to solve. Relevant sections of the documentation should resonate with you, depending on the persona whose goals you share. The documentation currently addresses the following personas:
+* [Architect]({{< relref "#architect">}})
+* [CorDapp Developer]({{< relref "#cordapp-developer">}})
+* [Corda Administrator]({{< relref "#corda-administrator">}})
+* [Network Operator]({{< relref "#network-operator">}})
+
+## Architect
+
+As an Architect considering or planning a project with Corda, you:
+* are curious about the benefits of adopting DLT technology, but also concerned about any risks.
+* want to know why DLT is a better fit for your business problem than a centralized solution.
+* are for looking for answers and guidance to the following questions:
+   * Who has access to my vault data and how is privacy preserved in Corda?
+   * How do I manage the compromising of cryptographic key material?
+   * How do I maintain compliance with GDPR (and other data protection regulations) if personal data is stored in Corda?
+   * How do I scale Corda?
+   * How do I maintain business continuity for Corda?
+   * How do I deploy Corda into my enterprise infrastructure and integrate it with my enterprise applications?
+
+As an Architect working with Corda, all sections of the Corda documentation are of interest to you, but particularly the [Key Concepts]({{< relref "../key-concepts/_index.md">}}).
+
+## CorDapp Developer
+
+A CorDapp Developer uses Corda to:
+* explore DLT and create their own chain of shared facts to experiment with.
+* design an enterprise production-grade distributed application.
+* work as part of a team to build an enterprise production-grade distributed application.
+* understand the software development lifecycle (SDLC) of a distributed application.
+* implement a method in the API to make something simpler.
+* discover best practice design tips for distributed applications.
+
+To get started developing CorDapps, see the [Developing Applications]({{< relref "../developing-applications/_index.md">}}) section.
+
+## Corda Administrator
+
+A Corda administrator is responsible for:
```

Comment:
```suggestion
A Corda Administrator is responsible for:
```
Not sure about capitalization strategy here. You capitalize "Architect" and both words in "CorDapp Developer".